.changeset/weak-schools-sit.md

```diff
@@ -0,0 +1,16 @@
+---
+'@leafygreen-ui/radio-box-group': patch
```

Comment:
These will all be `major` changes because they will break a lot of tests that check for the disabled attribute. 
It's also worth copying from this [changeset](https://github.com/mongodb/leafygreen-ui/blob/main/packages/button/CHANGELOG.md#major-changes-1).

Reply:
ahh gotcha, that make sense. thank you!